src/Clusterer.js

```diff
@@ -115,9 +115,9 @@ export class Clusterer extends React.Component {
     const { instance } = this.state;
 
     return (
-      <noscript>
+      <div>
```

Comment:
@mmarkelov can you apply the same change to Control element, please? I would do it myself, but this branch is in your fork and I don't have access to it 😅

Reply:
@gribnoysup sure!

Reply:
@gribnoysup I just leave div here to support react < 16

Reply:
@gribnoysup Should we place it inside Helpers.js?